applications/CompressiblePotentialFlowApplication/tests/test_CompressiblePotentialFlowApplication.py

```diff
@@ -5,8 +5,9 @@
 # Import Kratos "wrapper" for unittests
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
-# Import the tests o test_classes to create the suits
-from generalTests import KratosCompressiblePotentialFlowGeneralTests
+# Import the tests or test_classes to create the suits
+
+from test_potential_element_2d import TestPotentialElement2D
```

Comment:
is this still used?
didn't you implement this in C now?

Reply:
true, removed